src/main/scala/com/tapad/docker/DockerComposeKeys.scala

```diff
@@ -16,11 +16,12 @@ trait DockerComposeKeysLocal {
   val dockerMachineName = settingKey[String]("If running on OSX the name of the Docker Machine Virtual machine being used. If not overridden it is set to 'default'")
   val dockerImageCreationTask = taskKey[Any]("The sbt task used to create a Docker image. For sbt-docker this should be set to 'docker.value' for the sbt-native-packager this should be set to '(publishLocal in Docker).value'.")
   val testTagsToExecute = settingKey[String]("Set of ScalaTest Tags to execute when dockerComposeTest is run. Separate multiple tags by a comma. It defaults to executing all tests.")
+  val testExecutionExtraEnvironment = taskKey[Map[String, String]]("Additional ScalaTest Runner environment variables that will be passed into the ConfigMap.")
```

Comment:
Hi @sralmai, one other thing I was thinking about is that we should make the naming of new command more generic as it can be used to pass in additional configuration parameters other than just environment variables to the test cases.

I'd suggest something like `testExecutionExtraConfigTask` and an updating of the other variables and description text to align with the new naming of being more general.